Makefile

```diff
@@ -1,7 +1,6 @@
-CC = gcc
-AR = ar
-override CFLAGS := -std=gnu99 -D_FILE_OFFSET_BITS=64 -Wall -Wextra -O3 -g -MMD $(CFLAGS)
-LDFLAGS =
+CC ?= gcc
+AR ?= ar
```

Comment:
So the two options are either: remove the two lines completely, or use `?=` to stop the **makefile** from overriding user variables.